_episodes/02-loop.md

```diff
@@ -149,10 +149,12 @@ where each character (`char`) in the variable `word` is looped through and print
 The numbers in the diagram denote which loop cycle the character was printed in (1 being the first loop, and 6 being the final loop).
 
 We can call the [loop variable]({{ page.root }}/reference/#loop-variable) anything we like,
-but there must be a colon at the end of the line starting the loop,
-and we must indent anything we want to run inside the loop. Unlike many other languages, there is no
-command to signify the end of the loop body (e.g. `end for`); what is indented after the `for` statement belongs to the loop.
-
+but to open the block we can't forget to put a colon (`:`) at the end of the line starting the loop,
+then indent the code we want to run inside the loop. The colon identifies the beginning of a list of instructions, and the
+instruction before it tells us how to handle the list, in this case meaning it is a set of instructions we will loop over.
+Unlike many other languages, there is no
+command to signify the end of the loop body (e.g. `end for`); the code block (i.e. indented section) 
+after the `for` statement belongs to the loop -- you can also nest these!
```

Comment:
The reason I mention a code block, is that this is no-doubt a phrase they will hear in the real world of programming, and it is not a complicated on to grasp (as you see I defined it simply as the "indented section"). It's also not actually jargon - a "block of text" is a universal concept in English, that means the exact same thing in programming.

Also, the mention of nesting is an important concept so that they grasp, that they can string together complex logic in this way. Introducing these phrases do not considerably increase cognitive load, but plant seeds that will make taking to other lessons and self-guided instruction later more flexible and robust.

>"instructions before" don't tell us "how to handle the list" but tell Python how many times to repeat the loop and update loop variable, ...)

Colons are used for `if` statements as well, and the instructions before (i.e. `if ___` or `for ____`) tell us how to handle the instructions, whether to loop or conditionally run them... I think changing the phrase "list of instructions" to "set of instructions" would perhaps make it more clear, since your lessons don't seem to introduce Python `sets`, reducing the risk of overloading a word.